desktop/apps/auction/routes.js

```diff
@@ -28,13 +28,20 @@ export async function index (req, res, next) {
     res.locals.sd.AUCTION = sale
     fetchUser(req, res)
 
-    const [
-      { me },
-      { articles }
-    ] = await Promise.all([
-      metaphysics({ query: MeQuery(sale.id), req: req }),
-      metaphysics({ query: ArticlesQuery(sale._id) })
-    ])
+    const { me } = await metaphysics({
+      query: MeQuery(sale.id),
+      req: req
+    })
+
+    let articles = []
+
+    try {
+      ({ articles } = await metaphysics({
```

Comment:
BTW errors are aggregated into an array in GraphQL so you don't _have_ to split the request into like this, should just be able to `catch (errors)` and inspect that array.